blueprints/ember-cli-eslint/files/tests/.eslintrc

```diff
@@ -1,6 +1,5 @@
 {
-  "extends": "ember",
-  "rules": {
-    "no-console": 0
+  "env": {
+    "embertest": true
```

Comment:
it's allowing [these](https://github.com/sindresorhus/globals/blob/master/globals.json#L1199-L1212) globals